pkg/kubelet/cm/pod_container_manager_linux.go

```diff
@@ -77,7 +77,7 @@ func (m *podContainerManagerImpl) EnsureExists(pod *v1.Pod) error {
 	if !alreadyExists {
 		enforceCPULimits := m.enforceCPULimits
 		if utilfeature.DefaultFeatureGate.Enabled(kubefeatures.DisableCPUQuotaWithExclusiveCPUs) && m.podContainerManager.PodHasExclusiveCPUs(pod) {
-			klog.V(2).InfoS("Disabled CFS quota", "pod", klog.KObj(pod))
+			klog.V(5).InfoS("Disabled CFS quota", "pod", klog.KObj(pod))
```

Comment:
per discussion in https://github.com/kubernetes/kubernetes/pull/127525/files#r1940278408 this log is expected to happen rarely and is expected to tell us the fix took place. So I think it should be kept V=2 (my previous comment was too broad).